src/test/features/response/routes/defence-options.ts

```diff
@@ -6,6 +6,7 @@ import { attachDefaultHooks } from '../../../routes/hooks'
 import '../../../routes/expectations'
 import { checkAuthorizationGuards } from './checks/authorization-check'
 import { checkAlreadySubmittedGuard } from './checks/already-submitted-check'
+import { checkCountyCourtJudgmentRequestedGuardGuard } from './checks/ccj-requested-check'
```

Comment:
I cannot find `ccj-requested-check` anywhere in the committed files :(

Reply:
You're right. Added.